qa/tasks/cephfs/filesystem.py

```diff
@@ -1105,6 +1105,9 @@ def get_active_names(self, status=None):
         """
         return self.get_daemon_names("up:active", status=status)
 
+    def get_standby_replay_names(self, status=None):
```

Comment:
Can't the caller just invoke:

> self.fs.get_daemon_names('up:standby-replay')

Reply:
It is but this is even easier and we already have similar helper methods - 

```
    def get_active_names(self, status=None):
        """
        Return MDS daemon names of those daemons holding ranks
        in state up:active

        :return: list of strings like ['a', 'b'], sorted by rank
        """
        return self.get_daemon_names("up:active", status=status)
        ```
        
 Copied from here - https://github.com/ceph/ceph/blob/main/qa/tasks/cephfs/filesystem.py#L1100-L1107